tests/maintain/test_tag_edge.py

```diff
@@ -433,12 +433,16 @@ def test_same_tag_in_different_space(self):
         # 2.0 use space get from cache
         time.sleep(self.delay)
 
-        resp = self.client.execute_query('USE test_multi; CREATE Tag test_tag(); SHOW TAGS;')
+        resp = self.client.execute('USE test_multi;')
```

Comment:
why modify it?

Reply:
Be compatible with 1.0, DDL need check the is space choose.

Reply:
See line 85 in `test_permission.py`